src/Symfony/Component/Messenger/Bridge/Amqp/Transport/Connection.php

```diff
@@ -260,6 +266,11 @@ private static function normalizeQueueArguments(array $arguments): array
         return $arguments;
     }
 
+    private static function hasCaCertConfigured(array $amqpOptions): bool
+    {
+        return (isset($amqpOptions['cacert']) && '' !== $amqpOptions['cacert']) || 0 !== (int) ini_get('amqp.cacert');
```

Comment:
Why the casting of `ini_get('amqp.cacert')` to an int? What *is* `amqp.cacert` supposed to be - a path to the certificate?

Reply:
Reading documentation to answer you, I just say that ini_get can't read array (pdo.foo or amqp.cacert for eg..) I need another method so